Add draft content to the Type scale and pairing guideline page (#974)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/guidelines/type-pairing-and-spacing.mdx

---

<PageDescription>

Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before dive into creating an IBM.com website page.

</PageDescription>

<AnchorLinks>

<AnchorLink>Designing with type parings and spacings</AnchorLink>
<AnchorLink>Support</AnchorLink>

</AnchorLinks>

## Designing with type parings and spacings

Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.

> Below are some known type parings and spacings when designing for IBM.com to ensure you create clear hierarchies, proper information organization, and the best experience to guide users through the IBM.com experience.

Reviewer: Should we rephrase to "Below are some common type pairings" ? Thinking that these are the ones we have built into the components, "known" just felt a little odd when I read it.

Author: Good point.

> Below are some known type parings and spacings when designing for IBM.com to ensure you create clear hierarchies, proper information organization, and the best experience to guide users through the IBM.com experience.

Reviewer: ```suggestion
Below are some known type pairings and spacings when designing for IBM.com to ensure you create clear hierarchies, proper information organization, and the best experience to guide users through the IBM.com journey.
```

Reviewer: Seemed repetitive to say "experience" twice ?

<br />

### Content section

Example of Expressive-heading-02 pair with Body-long-02 at large breakpoints.

<Row>
<Column colMd={8} colLg={8}>

![Image of Table of contents component](../../images/guidelines/type-pairing-and-spacing/heading02_bodylong02.jpg)

</Column>
</Row>

**Tokens**

| `Type token` | `Breakpoint` | `info` |
| --- | --- | --- |
|`$expressive-heading-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.5em <br/>Weight: 600 / Semi-Bold <br/>Letter-spacing: 0px |

> |`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 22px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |

Reviewer: ```suggestion
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```

|`$expressive-heading-05`| Large |Type: IBM Plex Sans <br/>Size: 42px / 2.625rem <br/>Line-height: 50px / 3.125em <br/>Weight: 300 / Light <br/>Letter-spacing: 0px |
|`$expressive-heading-03`| Large |Type: IBM Plex Sans <br/>Size: 20px / 1.25rem <br/>Line-height: 28px / 1.75em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |

**Spacing**

| Breakpoint | `max` | `xlg` | `lg` | `md` | `sm` | 
| --- | --- | --- | --- | --- | --- |
| Margin top | `spacing-10` 64px | `spacing-10` 64px | `spacing-10` 64px | `spacing-07` 32px | `spacing-07` 32px |
| Margin bottom | `spacing-13` 160px | `spacing-13` 160px | `spacing-13` 160px | `spacing-12` 96px | `spacing-10` 64px |

<br />

### Content group

Example of Expressive-heading-04 pair with Body-long-02 at large breakpoints.

<Row>
<Column colMd={8} colLg={8}>

![Image of Table of contents component](../../images/guidelines/type-pairing-and-spacing/heading04_bodylong02.jpg)

</Column>
</Row>

**Tokens**

| `Type token` | `Breakpoint` | `info` |
| --- | --- | --- |
|`$expressive-heading-04`| Large |Type: IBM Plex Sans <br/>Size: 28px / 1.75rem <br/>Line-height: 36px / 2.25em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |

> |`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 22px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |

Reviewer: ```suggestion
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```

**Spacing**

| Breakpoint | `max` | `xlg` | `lg` | `md` | `sm` | 
| --- | --- | --- | --- | --- | --- |
| Margin top | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px |
| Margin bottom | `spacing-09` 48px | `spacing-09` 48px | `spacing-09` 48px | `spacing-09` 48px | `spacing-07` 32px |

<br />

### Content item

Example of Expressive-heading 02 pair with Body-long-02 at large breakpoints.

<Row>
<Column colMd={8} colLg={8}>

![Image of Table of contents component](../../images/guidelines/type-pairing-and-spacing/heading02_bodylong02.jpg)

</Column>
</Row>

**Tokens**

| `Type token` | `Breakpoint` | `info` |
| --- | --- | --- |
|`$expressive-heading-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.5em <br/>Weight: 600 / Semi-Bold <br/>Letter-spacing: 0px |

> |`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 22px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |

Reviewer: ```suggestion
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |
```

---
title: Type pairing and spacing
description: Typography can help create clear hierarchies, organize information, and guide users through a product or experience. Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.
---

<PageDescription>

> Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before dive into creating an IBM.com website page.

Reviewer: ```suggestion
Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before diving into creating an IBM.com experience.
```

> Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before dive into creating an IBM.com website page.

Reviewer: ```suggestion
Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before diving into creating an IBM.com experience.
```

</PageDescription>

<AnchorLinks>

<AnchorLink>Designing with type parings and spacings</AnchorLink>
<AnchorLink>Support</AnchorLink>

</AnchorLinks>

## Designing with type parings and spacings

> Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.

Reviewer: ```suggestion
Consistent type pairing and spacing can support different levels of architecture or varying levels of hierarchy, which are the keys to making the best IBM.com experience.
```

<br />

### Content item

Content item pairs `$expressive-heading-02` and `$body-long-02` with no space between the two type styles, the example below shows the large breakpoint.

<Row>
<Column colMd={8} colLg={8}>

![Image of Table of contents component](../../images/guidelines/type-pairing-and-spacing/heading02_bodylong02.jpg)

</Column>
</Row>

**Tokens**

| `Type token` | `Breakpoint` | `info` |
| --- | --- | --- |
|`$expressive-heading-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.5em <br/>Weight: 600 / Semi-Bold <br/>Letter-spacing: 0px |
|`$body-long-02`| Large |Type: IBM Plex Sans <br/>Size: 16px / 1rem <br/>Line-height: 24px / 1.375em <br/>Weight: 400 / Regular <br/>Letter-spacing: 0px |

**Spacing**

| Breakpoint | `max` | `xlg` | `lg` | `md` | `sm` | 
| --- | --- | --- | --- | --- | --- |
| Padding top | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px |
| Padding bottom | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px | `spacing-07` 32px |

<br />

## Support

> For any reason, if the pairings or spacings listed above do not meet your needs, please contact the Digital Design System team via the [#carbon-for-ibm-dotcom slack channel](https://cognitive-app.slack.com/archives/C2PLX8GQ6). 

Reviewer: ```suggestion
If for any reason, the type pairings or spacings listed above do not meet your needs, please contact the Digital Design System team via the [#carbon-for-ibm-dotcom slack channel](https://cognitive-app.slack.com/archives/C2PLX8GQ6). 
```
